drivers/usb/usbccgp/fdo.c

```diff
@@ -189,8 +189,14 @@ FDO_CreateChildPdo(
     }
 
     /* Create pdo for each function */
-    for(Index = 0; Index < FDODeviceExtension->FunctionDescriptorCount; Index++)
+    for (Index = 0; Index < FDODeviceExtension->FunctionDescriptorCount; Index++)
     {
+        if (FDODeviceExtension->FunctionDescriptor[Index].NumberOfInterfaces == 0)
+        {
+            // Ignore invalid devices
+            continue;
```

Comment:
It may be worth having a DPRINT1 to view these devices, for diagnostics purposes; what do you think about this?

Reply:
done